packages/container-registry-v5/package.json

```diff
@@ -13,14 +13,13 @@
     "repositoryPrefix": "<%- repo %>/blob/v<%- version %>/packages/container-registry-v5/<%- commandPath %>"
   },
   "dependencies": {
+    "@oclif/core": "^1.20.4",
```

Comment:
Not sure why this was added to dependencies, when the modules it's replacing were just dev dependencies?

Reply:
That is a good point. I can move it.